packages/std-contracts/src/test/Sample.t.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+// External
```

Comment:
What does this comment mean?

Reply:
Removed in [16e2033](https://github.com/latticexyz/mud/pull/264/commits/16e203374934ad3df1115076c0083939d69a1060)